src/worker/ingestion/process-event.ts

```diff
@@ -100,6 +100,8 @@ export class EventsProcessor {
             })
             try {
                 await this.handleIdentifyOrAlias(data['event'], properties, distinctId, teamId)
+            } catch (e) {
+                Sentry.captureException(e, { extra: { event: data } })
```

Comment:
Probably worth capturing teamId as well?

Reply:
That should come along with `data`. `data` is of type `PluginEvent`, which includes:

```js
export interface PluginEvent {
    distinct_id: string;
    ip: string | null;
    site_url: string;
    team_id: number;
    now: string;
    event: string;
    sent_at?: string;
    properties?: Properties;
    timestamp?: string;
    offset?: number;
    $set?: Properties;
    $set_once?: Properties;
    kafka_offset?: string;
    uuid?: string;
}
```